spec/rental_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../rental'
+
+describe Rental do
+  new_rental = Rental.new(
+    '2022-08-31',
+    'Da vinci code',
+    'John Doe'
+  )
+
+  describe 'View Rental' do
+    it 'show date of the rental' do
+      expect(new_rental.date).to eql '2022-08-31'
+    end
+  end
+end
```

Comment:
- [ ] OPTIONAL: Kudos on properly setting up testing. All tests are passing successfully. But I can see that you wrote one test per file. I strongly recommend you to think about the cases that could break the program and test them. Eg. test default values, empty values etc.

> Since there are no requirements that say test cases should be more than 1 this requirement is optional )

Reply:
thanks for your suggestions